compiler/z/codegen/OMRPeephole.cpp

```diff
@@ -958,7 +958,7 @@ OMR::Z::Peephole::tryToReduceAGI()
 bool
 OMR::Z::Peephole::tryToReduceCLRToCLRJ()
    {
-   if (!self()->comp()->target().cpu.getSupportsArch(TR::CPU::z10))
+   if (!self()->comp()->target().cpu.isAtLeast(OMR_PROCESSOR_S390_Z10))
```

Comment:
This change will conflict with #5481. Could you remove this piece from your PR and it will get cleaned up once #5481 is merged?

Reply:
#5481 is merged and I've removed this conflicting change now. Here's the new commit without any conflicts: f09371ba05dbb0ee3befbb0aced478291d2ea467